terraform/modules/hub/hub_saml_engine.tf

```diff
@@ -80,27 +92,92 @@ module "saml_engine" {
   certificate_arn            = var.wildcard_cert_arn
 }
 
+module "saml_engine_fargate" {
+  source = "./modules/ecs_fargate_app"
+
+  deployment = var.deployment
+  app        = "saml-engine"
+  domain     = local.root_domain
+  vpc_id     = aws_vpc.hub.id
+  lb_subnets = aws_subnet.internal.*.id
+  task_definition = templatefile("${path.module}/files/tasks/hub-saml-engine.json",
+    {
+      account_id                       = data.aws_caller_identity.account.account_id
+      deployment                       = var.deployment
+      domain                           = local.root_domain
+      optional_links                   = ""
+      image_identifier                 = "${local.tools_account_ecr_url_prefix}-verify-saml-engine@${var.hub_saml_engine_image_digest}"
+      nginx_image_identifier           = local.nginx_image_identifier
+      region                           = data.aws_region.region.id
+      location_blocks_base64           = base64encode(local.saml_engine_fargate_location_blocks)
+      redis_host                       = "rediss://${aws_elasticache_replication_group.saml_engine_replay_cache.primary_endpoint_address}:6379"
```

Comment:
Is this a typo?

```suggestion
      redis_host                       = "redis://${aws_elasticache_replication_group.saml_engine_replay_cache.primary_endpoint_address}:6379"
```

Reply:
No. `rediss` is correct here (it means "redis over TLS", or "secure redis"; "reidis-s", if you will).